teamsyncd/teamsync.cpp

```diff
@@ -160,11 +160,19 @@ int TeamSync::TeamPortSync::onChange()
         bool enabled;
 
         ifindex = team_get_port_ifindex(port);
-        team_ifindex2ifname(m_team, ifindex, ifname, MAX_IFNAME);
+
+        /* Skip if interface is not found */
+        if (!team_ifindex2ifname(m_team, ifindex, ifname, MAX_IFNAME))
+        {
+            SWSS_LOG_INFO("Interface ifindex:%d is not found", ifindex);
```

Comment:
%u

Reply:
team member interface ifindex(%u) cannot be found.

Reply:
updated